pkg/agent/events/watchtopic.go

```diff
@@ -290,15 +287,15 @@ func NewGovernanceAgentWatchTopic(name, scope string, agentResourceGroupKind v1.
 		{GroupKind: management.APIServiceInstanceGVK().GroupKind, ScopeName: scope, EventTypes: all},
 		{GroupKind: agentResourceGroupKind, ScopeName: scope, EventTypes: updated},
 	}
-	if features.IsMarketplaceSubsEnabled() {
-		kinds = append(kinds, []kindValues{
-			{GroupKind: management.CredentialGVK().GroupKind, ScopeName: scope, EventTypes: createdOrUpdated},
-			{GroupKind: management.AccessRequestGVK().GroupKind, ScopeName: scope, EventTypes: createdOrUpdated},
-			{GroupKind: management.ManagedApplicationGVK().GroupKind, ScopeName: scope, EventTypes: createdOrUpdated},
-			{GroupKind: management.CredentialRequestDefinitionGVK().GroupKind, ScopeName: scope, EventTypes: all},
-			{GroupKind: management.AccessRequestDefinitionGVK().GroupKind, ScopeName: scope, EventTypes: all},
-		}...)
-	}
+
+	kinds = append(kinds, []kindValues{
```

Comment:
same as above, maybe we should mark this as deprecated for future, as the GovernanceAgent does not exist